pybikes/data/whitebikes.json

```diff
@@ -0,0 +1,20 @@
+{
+    "instances":[
+       {
+            "tag":"whitebikes",
+            "meta":{
+                "latitude":48.149424,
+                "city":"Bratislava",
+                "name":"WhiteBikes",
+                "longitude":17.113922,
+                "country":"SK",
+                "company":[
+                    "Open Source Bike Share"
+                ]
```

Comment:
Once this `company` info is written on the python implementation, there will be no need to put it here.